packages/slider/src/component.tsx

```diff
@@ -68,6 +77,12 @@ export function Slider({ min = 0, max = 100, ...rest }: SliderProps) {
 
   const { handleKeyDown, handleFocus, handleBlur, handleMouseDown, handleClick, getThumbPosition, getThumbTransform, getShiftedChange } =
     createHandlers({ props: { min, max, ...rest }, sliderState });
+    handleFocus,
+    handleBlur,
+    handleMouseDown,
+    handleClick,
+    getThumbPosition,
+  } = createHandlers({ props: { min, max, ...rest }, sliderState });
```

Comment:
I think the code on lines 80-85 is a duplication of lines 78-79 and should be removed (remove line 80-85). :)

Reply:
I'm having problems with my linter, sorry about that:)